bokehjs/test/models/sources/ajax_data_source.coffee

```diff
@@ -0,0 +1,87 @@
+{expect} = require "chai"
```

Comment:
This is a new file, so it must be written in TypeScript.

Reply:
I was thinking about that. Putting aside that this was never a decision that was actually discussed or taken for all files, what is the specific value of *tests* being in TS? The biggest value of tests is that they be as quick and easy and painless to write as possible, so that more people are encouraged to write more of them. This class for instance has been untested, and has suffered as a result, for years. There is more value in there being tests at all, than not having them, regardless of language.

Reply:
recording decision to merge this now and update directories of files at time rather than mix sources together.